spec.html

```diff
@@ -31922,7 +31926,7 @@ <h1>Math.round ( _x_ )</h1>
         <p>It performs the following steps when called:</p>
         <emu-alg>
           1. Let _n_ be ? ToNumber(_x_).
-          1. If _n_ is *NaN*, *+&infin;*<sub>𝔽</sub>, *-&infin;*<sub>𝔽</sub>, or an integral Number, return _n_.
+          1. If _n_ is any of *NaN*, *+&infin;*<sub>𝔽</sub>, *-&infin;*<sub>𝔽</sub>, or an integral Number, return _n_.
```

Comment:
```suggestion
          1. If _n_ is any of *NaN*, *+&infin;*<sub>𝔽</sub>, or *-&infin;*<sub>𝔽</sub>, or _n_ is an integral Number, return _n_.
```
per above comment about it being weird to use "any of" when doing membership tests rather than testing identity.